apps/system/utils/fscmd.c

```diff
@@ -963,7 +991,22 @@ static int mount_handler(FAR const char *mountpoint, FAR struct statfs *statbuf,
 
 static int mount_show(foreach_mountpoint_t handler, FAR void *arg)
 {
-	return foreach_mountpoint(handler, arg);
+	int fd;
+	int ret;
+	struct foreach_mountpoint_s info;
+
+	fd = open(FSCMD_DRVPATH, O_RDWR);
+	if (fd < 0) {
+		FSCMD_OUTPUT(CMD_FAILED, "open", FSCMD_DRVPATH);
+	}
+
+	info.handler = handler;
+	info.arg = arg;
+
+	ret = ioctl(fd, FCIOC_FOREACH_MOUNTPOINT, (unsigned long)&info);
```

Comment:
Exception handling is required for invalid fd.

Reply:
I will modify

return ERROR